tests/unit/tasks_test.py

```diff
@@ -84,23 +84,19 @@ def test_classification_no_label_column_in_input():
 def test_classification_no_label_column_in_config():
```

Comment:
I think this test itself can be deleted. it doesn't make sense to set it to nothing